pages/documents/MessagingChannels/ProactiveMessaging/overview.md

```diff
@@ -0,0 +1,253 @@
+---
```

Comment:
@EdenKupe 

Proactive team has finished review and created a pull request to add the documentation to the LP developer community. Can you review and merge it and let us know when it is to be published and appear to the public?

Reply:
@lmartliveperson

Reply:
@ppatel-lp Merged, please review and verify